.circleci/config.yml

```diff
@@ -230,14 +230,24 @@ workflows:
           filters:
             tags:
               only: /^v.*/
+      - architect/push-to-docker:
+          username: "${QUAY_USERNAME}"
+          password: "${QUAY_PASSWORD}"
```

Comment:
Why do we need to pass that when they are read from env var in the job?

Reply:
Passing those values here make it clear and explicit which values are used by the orb, instead of reading env vars from within the orb.

Reply:
We decided to pass env var names instead of the value, for safety (password value won't ever be exposed).